src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -1202,6 +1203,26 @@ public static void TestingStringsConversionToGuid(string testString, string expe
             Assert.Equal(json.BytesConsumed, json.CurrentState.BytesConsumed);
         }
 
+        [Theory]
+        [MemberData(nameof(JsonGuidTestData.ValidGuidTests), MemberType = typeof(JsonGuidTestData))]
+        public static void TryGetGuid_HasValueSequence_RetrievesGuid(string testString, string expectedStr)
```

Comment:
nit: expectedStr => expectedString for consitency (same for TestingStringsConversionToGuid)